src/components/VMapboxStylePicker.vue

```diff
@@ -41,19 +54,37 @@ export default {
       // initialize control
       this.map.addControl(this, 'bottom-right')
 
+      // creating the variable
+      var year = 2016
+
+      // moment and evetbus connection
+      bus.$on('slider-update', (date) => {
+        var b = date.enddate.slice(3)
```

Comment:
DO not slice this! use the whole date in moment!! If ever we decide to change the format in enddate this will go wrong.

Always use descriptive variable names (not 'b')

Reply:
i have change it to => var endtime = date.enddate

Reply:
enddate has become: const enddate = moment([endtime], 'MM-YYYY'). format('YYYY')